qa/tasks/cephfs/test_acls.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from StringIO import StringIO
+
+from xfstests_dev import XFSTestsDev
+
+log = logging.getLogger(__name__)
+
+class TestACLs(XFSTestsDev):
+
+    def test_acls(self):
+        from tasks.cephfs.fuse_mount import FuseMount
+        from tasks.cephfs.kernel_mount import KernelMount
+
+        if isinstance(self.mount_a, FuseMount):
+            log.info('client is fuse mounted')
+        elif isinstance(self.mount_a, KernelMount):
+            log.info('client is kernel mounted')
+
+        self.mount_a.client_remote.run(args=['sudo', './check',
+            'generic/099'], cwd=self.repo_path, stdout=StringIO(),
+            stderr=StringIO(), timeout=30, check_status=True,
+            label='running tests for ACLs from xfstests-dev')
```

Comment:
Huh...now that we have all of this infrastructure here, I think we ought to expand this to allow for a full xfstests run. It seems a shame to do all of this setup just to run a single test.

That said, a few tests are known to fail on cephfs (at least for now), so I think we'd need to either whitelist the failures or skip them.

That can be done in a later PR though. I'd suggest that we just merge this as an ACL test only for now and look at doing that later.

Reply:
> That can be done in a later PR though. I'd suggest that we just merge this as an ACL test only for now and look at doing that later.

Agreed. I'll create a PR that'll do that as soon as this one merges.